api/v1beta1/keystoneapi_types.go

```diff
@@ -221,7 +221,7 @@ type KeystoneAPIStatus struct {
 	Hash map[string]string `json:"hash,omitempty"`
 
 	// API endpoint
-	APIEndpoints map[string]string `json:"apiEndpoint,omitempty"`
+	APIEndpoints map[string]string `json:"apiEndpoints,omitempty"`
```

Comment:
Yes, I agree that the JSON/YAML field name should match the plurality of the Golang struct field name